andela/bp-andela-media
# #161551230 Add docker to deployment

#### What does this PR do?
Adds a docker deployment strategy

#### Description of Task to be completed?
- Add dockerfile
- Add docker-compose.yml

#### How should this be manually tested?

**PS** ensure you have docker installed.

Run the following commands.
1. `$ docker-compose build web`
2. `$ docker-compose run web rake db:create`
3. `$ docker-compose up web`
4. Visit [localhost](http://localhost:3000) to see the default welcome page for Rails.

Follow [this tutorial](https://docs.docker.com/compose/rails/) for an explanation of what the changes are supposed to do.

#### What are the relevant pivotal tracker stories?
[#161551230](https://www.pivotaltracker.com/story/show/161551230)

#### Any background context you want to add?
- The application is currently running on Heroku as a staging to the production pipeline


#### Screenshots
